.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+    "presets": [
+      [
+        "env",
+        {
+          "targets": {
+            "browsers": ["last 2 versions", "ie >= 7"]
```

Comment:
it feels a little odd putting "browsers" here, even though it should satisfy the requirement to transpile for es2015... maybe we can just say node 4.x? I believe @chfritz set up the initial transpilation for 4.5